### DISCUSSION TITLE
How could I set GITHUB_OAUTH environment

### DISCUSSION THREAD

**Discussion by xngwu1995** [hub4j/github-api#1313] [Q&A]
Hey Guys,

Anyone could tell me how could I set GITHUB_OAUTH environmental variables.
![image](https://user-images.githubusercontent.com/62212729/142746812-5dfc4bc0-1d11-45da-9772-e21fb7c1b3d8.png)

**Comment by bitwiseman** [comment]
`export GITHUB_OAUTH=<your key>`
